src/test/compile-fail/task-rng-isnt-sendable.rs

```diff
@@ -10,10 +10,10 @@
 
 // ensure that the ThreadRng isn't/doesn't become accidentally sendable.
 
-use std::rand; //~ ERROR: module `rand` is private
+use std::__rand::ThreadRng;
```

Comment:
The change to this test is orthogonal to the rest of the PR -- c.f. https://github.com/rust-lang/rust/pull/35776#discussion_r75729787